src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/NullableAttributes.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why do you need this file? Is it just for the `MaybeNullWhenAttribute` case?

Reply:
Yeah. It used to also be for `NotNull`, but once I started using `List`, that went away.
On net472, we don't have access to these attributes, so this file is a fallback.
I can see these attributes being used again in the future, so I think it's worth keeping.